wordlist.txt

```diff
@@ -26,8 +26,8 @@ decrypt
 dereference
 deserialized
 devp
-devp2p
 DevP
```

Comment:
Whats DevP

Reply:
It's DevP from DevP2P because the spellchecker oddly breaks at numbers sometimes